client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -246,8 +247,14 @@ const siteSetupFlow: Flow = {
 
 			navigate( 'processing' );
 
-			// Clean-up the store so that if onboard for new site will be launched it will be launched with no preselected values
-			resetOnboardStoreWithSkipFlags( [ 'skipPendingAction', 'skipIntent', 'skipSelectedDesign' ] );
+			if ( ! isGoalsAtFrontExperiment ) {
+				// Clean-up the store so that if onboard for new site will be launched it will be launched with no preselected values
+				resetOnboardStoreWithSkipFlags( [
+					'skipPendingAction',
+					'skipIntent',
+					'skipSelectedDesign',
```

Comment:
`skipSelectedDesign` is apart of the experiment. It was added so that we can use it to show the launchpad task `select a design` when a paid theme was switched to free [here](https://github.com/Automattic/wp-calypso/pull/97539).